sky/packages/sky/lib/src/widgets/navigator.dart

```diff
@@ -177,7 +177,7 @@ abstract class Route {
     _performance = createPerformance();
   }
 
-  PerformanceView get performance => _performance?.view;
```

Comment:
This is too powerful.

As per our discussion, I think PerformanceView should expose an "obtainWriteHandle" method (or whatever) that gives you a handle to modify the performance, but which can be revoked. When the current route is not this route or the route below it (the stock home, e.g.) then you can't obtain a handle. When navigator pops or pushes, you lose your handle. When another handle is obtain, you lose your handle.